### ISSUE TITLE
Softban cause by not recycling item

### ISSUE THREAD

**Issue by 4triX** [Necrobot-Private/NecroBot#376]
### Expected Behavior:
Bot recycle items before it reach the bag limit.

### Actual Behavior:
Bot does not recycle items, and then Pokestop cant be looted. It shows SOFTBAN error. However, catching is working fine

After checking on the inventory, it shows 351/350 items.

### Your config.json:
```
  "RecycleConfig": {
    "VerboseRecycling": false,
    "RecycleInventoryAtUsagePercentage": 90.0,
    "RandomizeRecycle":true,
    "RandomRecycleValue": 5,
    "TotalAmountOfPokeballsToKeep": 90,
    "TotalAmountOfPotionsToKeep": 30,
    "TotalAmountOfRevivesToKeep": 40,
    "TotalAmountOfBerriesToKeep": 50,
    "MaxPokeballsToKeep": 50
  },
```

### Other Information
Screen shot : http://imgur.com/a/3gnn3

Version: 1.0.0.23
Windows Version: Windows server 2016
What is affected (catch, transfer, favorite, GPX, evolve, recycle, snipe, human walk snipe, pokestops, powerup/levelup, other): Recycle, Pokestop

**Comment by XurWuy** [comment]
Not just items, but if your pokemon bag full also resulted in softban wjile looting pokestop....

It's like either pokebag full or items full, both will resulted in softban.

**Comment by samuraitruong** [comment]
pokemon full is other story , working on this item . seem data return from server has changed.

**Comment by samuraitruong** [comment]
Fixed #377
